src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -9239,10 +9239,10 @@ namespace WaterThermalTanks {
 			if ( WaterThermalTank( WaterThermalTankNum ).FirstRecoveryDone && WaterThermalTank( WaterThermalTankNum ).FirstRecoveryFuel > 0.0 ) {
 				// Calculate Recovery Efficiency based on energy used to recover from the first draw
 				// FirstRecoveryFuel is recorded inside the CalcWaterThermalTank subroutine
-				RecoveryEfficiency = DrawMass * CPHW( 57.2222 ) * ( 57.2222 - 14.4444 ) / WaterThermalTank( WaterThermalTankNum ).FirstRecoveryFuel;
+				RecoveryEfficiency = DrawMass * CPHW() * ( 57.2222 - 14.4444 ) / WaterThermalTank( WaterThermalTankNum ).FirstRecoveryFuel;
 
 				// Calculate Energy Factor based on total energy (including parasitics) used over entire test
-				EnergyFactor = TotalDrawMass * CPHW( 57.2222 ) * ( 57.2222 - 14.4444 ) / FuelEnergy;
+				EnergyFactor = TotalDrawMass * CPHW() * ( 57.2222 - 14.4444 ) / FuelEnergy;
```

Comment:
@rraustad That's a good point.  Luckily we'll have the history to go back and correct them, should we ever do it.